react/OrderPlaced.js

```diff
@@ -3,13 +3,13 @@ import PropTypes from 'prop-types'
 import { compose, graphql } from 'react-apollo'
 import { branch, renderComponent } from 'recompose'
 import { Loader } from 'vtex.order-details'
+import { withRuntimeContext } from 'vtex.render-runtime'
 import Header from './components/Header'
```

Comment:
Separate blocks of codes with an empty line

Reply:
Updated :)